README.md

```diff
@@ -83,17 +84,18 @@ module "es_alarms" {
 |-----------------------------------------------|-------------|:----:|:-------:|:--------:|
```

Comment:
Can you also help to align the `|` for the first column after we adding/removing all the variables?

Reply:
Done, pushing shortly

Reply:
Done in latest push